source/macos/AdaptiveCards/AdaptiveCards/Views/ACRDateField.swift

```diff
@@ -141,6 +142,11 @@ class ACRDateField: NSView, InputHandlingViewProtocol {
         iconButton.centerYAnchor.constraint(equalTo: centerYAnchor).isActive = true
     }
     
+    private func setupPopover() {
+        datePickerCalendar.dateValue = Date()
+        datePickerTextfield.dateValue = Date()
+    }
```

Comment:
Just to confirm, if there is a default value provided, the picker show the provided value right? Not the current date?

Reply:
A UT for this will be great!